money/wages.qmd

```diff
@@ -0,0 +1,111 @@
+
+This dataset can help us build econometric models that predict wages based on a host of worker-level information. You can also predict binary outcomes like union membership. While understanding relationships between variables is itself valuable, it is important to assess how causality is involved. For example, if married people tend to have higher wages controlling for other demographic variables, is this a causal relationship? If it's causal, what is the direction of causality? (For example, do higher wages make people want to "settle down" and get married, or do married people demand companies pay them more?) Or, is there *endogeneity* (two-way causality) in this relationship? These are some of the questions economists ask.
+
+
+## Data
+
+
+This dataset describes worker-level hourly wages from the March 2013 U.S. Current Population Survey (CPS). Each row represents a worker's response to the CPS. Columns include worker demographic traits such as gender, race, age, and education level. 
+
+There are 9799 rows; each row is an independent individual worker. There is no missing data.
+
+
+
+### Data preview
+
+```{r, echo=FALSE, results="asis"}
+source("../preview_dataset.R")
+preview_datasets()
+```
+
+### Variable descriptions
+
+All information is self-reported through the Current Population Survey (CPS).
+
+| Variable | Description |
+|----|-------------|
+| wage | hourly wage (March 2013 US dollars) |
+| hrswork | hours worked last week |
+| age | age |
+| exper | potential experience = age - educ - 6 |
+| educ | years of education |
+| female | =1 if female |
+| asian | =1 if asian |
+| black | =1 if black |
+| white | =1 if white |
+| divorced | =1 if divorced |
+| married | =1 if married |
+| single | =1 if single (never married) |
+| nchild | number of own children in household |
+| faminc | gross yearly income of other family members (March 2013 US dollars) |
+| insure | =1 if covered by private health insurance |
+| mcaid | =1 if covered by Medicaid last year |
+| mcare | =1 if covered by Medicare last year |
+| union | =1 if a union member |
+| metro | =1 if in metropolitan area |
+| northeast | =1 if in northeast region |
+| midwest | =1 if in midwest region |
+| south | =1 if in south region |
+| west | =1 if in west region |
+
+
+
+
+## Questions
+
+Suppose you would like to learn about demographic characteristics that are associated with higher wages. Suppose you want run multivariate regression using OLS on worker wages.
+
+1. If the outcome you're interested in is `wage`, should you perform any transformations? (Hint: plot a histogram of wage - does it appear symmetrical and bell-shaped?).
+
+2. What demographic variables would you include as covariates? Should there be any interaction terms?

```

Comment:
In principle, if "you would like to learn about demographic characteristics that are associated with higher wages", isn't the answer "all of them" and "yes"? This might benefit from a more pointed research question, either one posed causally (you want to know what *causes* higher wages) or maybe one targeted at a specific variable (do union members get paid more?).